src/cf-tables/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-tables",
-  "version": "1.1.0",
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/396/commits/5ec3a501b3faa2a5060d58777d15bcf79568b265.